db/migrate/20221030181020_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    change_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :course, null: false, foreign_key: true
+      t.text :body
```

Comment:
Will we be extending this with types (note, question, etc) and role based visibility (private, public posts) later?

Reply:
As far as I remember, the professor told us to ignore these stuff (like post types, etc.) for the first versions and If we had extra time later, add them.

Reply:
Ah, good point, thanks for clarifying! Will resolve the conversation.